newIDE/app/src/Export/GenericExporters/OnlineWebExport/OnlineGamePropertiesDialog.js

```diff
@@ -92,6 +87,11 @@ export const OnlineGamePropertiesDialog = ({
         orientation: orientation || 'default',
       })
     ) {
+      // If the project has been modified, then save it.
```

Comment:
This fixes the bug where the project was saved before the modifications were applied. So it was useless........